editor/animation_track_editor.cpp

```diff
@@ -3706,7 +3706,7 @@ void AnimationTrackEditor::_animation_track_remove_request(int p_track, Ref<Anim
 	int idx = p_track;
 	if (idx >= 0 && idx < p_from_animation->get_track_count()) {
 		EditorUndoRedoManager *undo_redo = EditorUndoRedoManager::get_singleton();
-		undo_redo->create_action(TTR("Remove Anim Track"), UndoRedo::MERGE_DISABLE, p_from_animation.ptr());
+		undo_redo->create_action(TTR("Remove Anim Track"), UndoRedo::MERGE_DISABLE, p_from_animation.ptr(), true);
```

Comment:
To whomever ends up reviewing this, and wonders about the changes to `_animation_track_remove_request`: I couldn't figure out how to compose this undo/redo action together with the new `_convert_track_to_bezier` without resorting to using this new `p_backward_undo_ops` at the top-most action in `EDIT_CONVERT_TO_BEZIER_CONFIRM`. This seemed to then necessitate using that `p_backward_undo_ops` flag here as well, since the list is reversed at the actual commit (i.e. the top-most action) rather than per sub-action.

So the changes to this method are largely just about reversing the list of undo actions.

If anyone can see a way of doing this while leaving `_animation_track_remove_request` unchanged I'm all ears.